exchange/websocket/manager.go

```diff
@@ -1072,13 +1103,14 @@ func (m *Manager) GetConnection(messageFilter any) (Connection, error) {
 		return nil, ErrNotConnected
 	}
 
-	for _, wrapper := range m.connectionManager {
-		if wrapper.setup.MessageFilter == messageFilter {
-			if wrapper.connection == nil {
-				return nil, fmt.Errorf("%s: %s %w associated with message filter: '%v'", m.exchangeName, wrapper.setup.URL, ErrNotConnected, messageFilter)
-			}
-			return wrapper.connection, nil
+	for _, ws := range m.connectionManager {
+		if ws.setup.MessageFilter != messageFilter {
```

Comment:
Semi OOS: Petition to change MessageFilter to `EndPointID`. Its definitely not a connection identifier as we now have a slice of them per endpoint. As you say in your own words, it doesn't filter a websocket message